src/classlibnative/bcltype/number.cpp

```diff
@@ -134,8 +135,191 @@ unsigned int Int64DivMod1E9(unsigned __int64* value)
 
 #else // _TARGET_X86_ && !FEATURE_PAL
 
-#pragma warning(disable:4273)
-extern "C" char* __cdecl _ecvt(double, int, int*, int*);
+void ecvt( double value, int count, int* dec, int* sign, wchar_t* digits )
```

Comment:
That is only the 32-bit implementation which does not hit this particular code path.

This particular _ecvt declaration should be hit if:
* `_TARGET_X86_` is not defined
or
* `_TARGET_X86_` and `FEATURE_PAL` are both defined

So 64-bit windows and all non-windows platforms.

There still might be some code I'm missing somewhere, but we should make sure that is the case. We likely don't want to replace the windows implementation where the code is likely still faster than the replacement here.